.jazzy.yaml

```diff
@@ -0,0 +1,15 @@
+module: KituraTemplateEngine
+author: IBM
+github_url: https://github.com/IBM-Swift/Kitura-TemplateEngine/
+
+theme: fullwidth
+clean: true
+exclude: [Packages]
+
+readme: README.md
+
+skip_undocumented: false
+hide_documentation_coverage: false
+
+xcodebuild_arguments: [-project, Kitura-TemplateEngine.xcodeproj, -target, KituraTemplateEngine, -toolchain, org.swift.3020160912a, LIBRARY_SEARCH_PATHS=.build/debug]
```

Comment:
Now that you can use Swift 3.0 to run Jazzy, this should be:

`xcodebuild_arguments: [-project, Kitura-TemplateEngine.xcodeproj, -target, KituraTemplateEngine, LIBRARY_SEARCH_PATHS=.build/debug]`